src/platform/packages/shared/kbn-coloring/src/shared_components/color_mapping/config/utils.ts

```diff
@@ -16,3 +17,27 @@ export function isCategoricalColorConfig(config: Config): config is CategoricalC
 export function isGradientColorConfig(config: Config): config is GradientConfig {
   return config.colorMode.type === 'gradient';
 }
+
+export function getOtherAssignmentColor(
+  specialAssignments: Config['specialAssignments'],
+  assignments: Config['assignments']
+):
```

Comment:
So is your plan to remove this function when we add a CM transform to fix the config?

Reply:
if we can have a runtime migration that converts empty assigmment to have always and only the loop configuration it will be better